astropy/time/tests/test_basic.py

```diff
@@ -867,51 +867,62 @@ def test_explicit_example(self):
         assert t.to_value('mjd', subfmt='str') == '54321.000000000001'
         assert t.to_value('mjd', 'bytes') == b'54321.000000000001'
         expected_long = np.longdouble(54321.) + np.longdouble(1e-12)
-        assert t.to_value('mjd', subfmt='long') == expected_long
+        # Check we're the same to within the double holding jd2
+        # (which is less precise than longdouble on arm64).
+        assert np.allclose(t.to_value('mjd', subfmt='long'),
+                           expected_long, rtol=0, atol=2.**-52)
```

Comment:
64-bits are filtered out, but 80-bit would still work, since the setting of `jd2` from it is exact to 2**-52, i.e., the same rounding errors will be made. (And other pieces test that the input longs are different)